packages/eslint-plugin/tests/rules/array-type.test.ts

```diff
@@ -1955,6 +1955,44 @@ interface FooInterface {
       options: [{ default: 'array-simple' }],
       output: 'declare function foo<E extends readonly string[]>(extra: E): E;',
     },
+    {
+      // https://github.com/typescript-eslint/typescript-eslint/issues/10519
```

Comment:
[Nit]: I think we can omit comments linking back to the original issue (There also two more comments at the tests below).

Reply:
Done!